diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x 'Makefile.in*' -x configure -x gnome-doc-utils.make -x intltool.m4 -x omf.make -x xmldocs.make origsrc/libieee1284-0.2.10/Makefile.am src/libieee1284-0.2.10/Makefile.am
--- origsrc/libieee1284-0.2.10/Makefile.am	2005-01-21 08:33:54.000000000 -0600
+++ src/libieee1284-0.2.10/Makefile.am	2006-07-26 18:09:45.324250000 -0500
@@ -9,6 +9,7 @@
 # When rolling a release, remember to adjust the version info.
 # It's current:release:age.
 libieee1284_la_LDFLAGS = -version-info 5:2:2 \
+	-no-undefined \
 	-export-symbols $(top_srcdir)/libieee1284.sym
 include_HEADERS = include/ieee1284.h
 INCLUDES = -I$(top_srcdir)/include
@@ -34,9 +35,9 @@
 src_ieee1284module_la_SOURCES = src/ieee1284module.c
 if PYTHON
 pyexec_LTLIBRARIES = src/ieee1284module.la
-src_ieee1284module_la_LDFLAGS = -module -avoid-version
-src_ieee1284module_la_LIBADD = libieee1284.la
-src_ieee1284module_la_CFLAGS = -I/usr/include/python$(PYTHON_VERSION) -fpic
+src_ieee1284module_la_LDFLAGS = -module -avoid-version -no-undefined
+src_ieee1284module_la_LIBADD = libieee1284.la -L/usr/lib/python$(PYTHON_VERSION)/config -lpython$(PYTHON_VERSION)
+src_ieee1284module_la_CFLAGS = -I/usr/include/python$(PYTHON_VERSION)
 endif
 
 man3_MANS = doc/libieee1284.3 \
